src/components/PersonalSettings.vue

```diff
@@ -147,9 +165,46 @@ export default {
 			documentSigningCert: this.initial.documentSigningCert || '',
 			documentSigningKey: this.initial.documentSigningKey || '',
 			documentSigningCa: this.initial.documentSigningCa || '',
+			tokenGenerated: false,
+			accessToken: '',
+			accessTokenTTL: '',
+			userId: '',
```

Comment:
```suggestion
			userId: getCurrentUser()?.uid,
```

This is never changing and ready on page load. We don't need the rather complex chek logic in mounted hook.

Reply:
Done